hca/upload/api_client.py

```diff
@@ -0,0 +1,15 @@
+import requests
+
+
+class ApiClient:
+
+    UPLOAD_SERVICE_API_URL_TEMPLATE = "https://upload.{deployment_stage}.data.humancellatlas.org/v1"
+
+    def __init__(self, deployment_stage):
+        self.api_url_base = self.UPLOAD_SERVICE_API_URL_TEMPLATE.format(deployment_stage=deployment_stage)
```

Comment:
You can also do `ApiClient.UPLOAD_SERVICE_API_URL_TEMPLATE`.  I somewhat prefer that b/c it is not really meant to be specific to the _instance_.  But not a big deal either way.